pkg/crypt/crypt_test.go

```diff
@@ -62,3 +62,9 @@ func TestGenSalt(t *testing.T) {
 	retSaltSecond, _ := genSalt(length)
 	assert.NotEqual(t, retSaltFirst, retSaltSecond)
 }
+
+func TestCryptItself(t *testing.T) {
+	cryptedPass, err := crypt("Hello world!", "$6$saltstring")
```

Comment:
test vector is taken from the glibc source (e.g. https://github.com/lattera/glibc/blob/master/crypt/sha512c-test.c#L12)